docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -325,7 +325,7 @@ replacing the host IP with your host address:
 ## Add Device Service to docker-compose File (docker-compose.yml)
 
 Download the `Geneva` release docker-compose file from
-<https://github.com/edgexfoundry/developer-scripts/blob/master/releases/geneva/compose-files/docker-compose-geneva-redis.yml>.
+<https://github.com/edgexfoundry/edgex-compose/blob/geneva/docker-compose-geneva-redis.yml>.
```

Comment:
Note that this link will be correct once PR for Geneva branch is merged. Updating this example to Hanoi is out of scope.